### DISCUSSION TITLE
Roadmap and sustainability

### DISCUSSION THREAD

**Discussion by newtechfellas** [tamagui/tamagui#699] [General]
My team is checking native base vs tamagui. While tamagui features are impressive, wondering about future roadmap and support.

Is this project one man show? 99% (?) of the changes seems to be implemented by Nate. Most of the issues are also addressed by him. Wondering if this is single point of failure for future.

I understand this repo is open source and relies on community contribution. However, checking if there is a dedicated team as well from Nate to support the library?

This question by no means undermines the excellent features of this library. Its merely to make a decision to use the library for a multi year product consideration and tamagui is relatively new.

**Comment by jdordoigne** [comment]
We were in the same process and decided not go with nativebase. Check this link : https://github.com/GeekyAnts/NativeBase/issues/4302
They are in the process of rewriting everything to a new system called dank (https://dank.style/), which is in alpha.

**Comment by rajeevnodzon** [comment]
This library is brilliant. But, Nate we don't see you replying back to discussions, which is a concern.

**Comment by cerpinn** [comment]
To address your concern about tamagui being a one man show, it does have a good number of active contributors, but also, I'll be joining to help Nate (full time) in a few days! 
Will probably take a bit to get the hang of the lower level parts of the codebase but will definitely help with github discussions, issues and discord support.
There are plans for a public roadmap too!
